docs/architecture/eth_sol_solution.md

```diff
@@ -135,22 +135,23 @@ Fig. 2
 
 </div>
 
-To perform this transaction, the Neon EVM emulator makes a request to Solana to get state data. The proxy requests a blockchain state from Solana and makes a test launch of N-tx using the Solana state. On the basis of the data received, Neon Proxy forms the new transaction according to Solana's rules. The transaction generated for Solana will contain the packaged N-tx, as well as the account data required to complete the transaction, including:
-  * The payer: the account that pays for the transaction
-  * Other accounts involved in the execution of this transaction
+To perform this transaction, the Neon EVM emulator makes a request to Solana to get state data. The proxy requests a blockchain state from Solana and makes a test launch of the N-tx using the Solana state. On the basis of the data received, Neon Proxy forms the new transaction according to Solana's rules. The transaction generated for Solana will contain the packaged N-tx, as well as the account data required to complete the transaction, including:
 
-Next, the transaction is forwarded inside Solana, where participants of the transaction are determined according to the account data. The data added includes details about people who initiate and take part in processing the transaction. The operator of the proxy is specified as the payer. The payer will be rewarded for completing the transaction.  
+- The payer: the account that pays for the transaction
+- Other accounts involved in the execution of this transaction
 
-Next, the transaction is transferred to Neon EVM, where its signature is checked according to Ethereum rules. If the signature is valid, the transaction will be transferred to the Solana blockchain for execution. Since testing of N-tx is carried out, it is possible to provide Solana with all the necessary information to run the execution of the N-tx in parallel.  
+This transaction is then forwarded inside Solana, which uses the account data to determine the participants in the transaction. This data includes details about people who initiate and take part in processing the transaction. The operator of the proxy is specified as the payer. The payer will be rewarded for completing the transaction.
```

Comment:
@YYYurchenko I will leave approval to you. NB this is the only section that requires review